docs/source/user/api.rst

```diff
@@ -656,6 +656,24 @@ in the URL.
 
 Responds only to **GET**.
 
+.. _radius_usage_api_view:
+
+User Radius Usage
+-----------------
+
+**Requires the user auth token (Bearer Token)**.
+
+Returns the radius usage of the logged-in user and the organization specified
+in the URL.
+
+It executes the relevant RADIUS counters and returns the exhausted values.
```

Comment:
I think "the exhausted values" may not be very clear to every user.

Something like the following may work better, feel free to improve upon my suggestion.

```suggestion
It executes the relevant RADIUS counters and returns information that allows to know how much time and/or traffic the user has left.
```